data_steward/cdr_cleaner/manual_cleaning_rules/dc732_suppress.py

```diff
@@ -0,0 +1,598 @@
+# # DC-732 Retract Rows with Suppressed Concepts
+#
+# Identify, log and remove all rows from all tables in specified deid datasets where at least one
+# concept_id field references a concept which must be suppressed.
+#
+# ## Notes
+#
+#  * Prior to running the table identified by `CONCEPT_LOOKUP_TABLE` must be loaded with concepts to
+#    suppress. At minimum this table should have the field `concept_id`.
+#  * This can safely be re-run (e.g. if a new concept is added to the lookup).
+
+import argparse
+
+import jinja2
+import pandas as pd
+from google.cloud import bigquery
+
+from resources import DC732_CONCEPT_LOOKUP_CSV_PATH
+from utils import bq
+
+# -
+
+ISSUE_NUMBER = 'DC-732'
+ISSUE_PREFIX = 'dc732_'
+ROWS_RESOURCE_NAME = f'{ISSUE_PREFIX}suppress_rows'
+SUMMARY_RESOURCE_NAME = f'{ISSUE_PREFIX}suppress_summary'
+
+JINJA_ENV = jinja2.Environment(
+    # block tags on their own lines
```

Comment:
does codacy ever mention using the `autoescape=True` parameter with this environment to help prevent cross-site scripting attacks?

Reply:
Yes, [it does](https://app.codacy.com/manual/pmi-ops-org/curation/pullRequest?prid=5314486). Thanks for bringing to my attention. I will fix. Currently wondering when codacy stopped being one of our pull request checks so I might've caught this myself... 🤔

Reply:
It's weird.  Codacy alerted me to it in the integration tests when I started using jinja2.Environment.  So I don't know why it would have missed this one.  Not sure how it's set up right now.

Reply:
It's weird.  Codacy alerted me to it in the integration tests when I started using jinja2.Environment.  So I don't know why it would have missed this one.  Not sure how it's set up right now.

Reply:
_I think_ it's resolved... does it look ok to you?